### PR TITLE
Topic/ud progress

### PR DESCRIPTION
**Pull request by alex-mikheev**
@yosefe  pls take a look

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/856/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/283/ for details. (Mellanox internal link)

**Comment by shamisp** [comment]
autobot build

File: src/uct/ib/ud/verbs/ud_verbs.c [hunk 1]
```diff
@@ -97,75 +86,63 @@ static inline void uct_ud_verbs_iface_tx_inl(uct_ud_verbs_iface_t *iface, uct_ud
...
-    ucs_status_t status;
+    uct_ud_verbs_iface_t *iface = ucs_derived_of(ud_iface,
+                                                 uct_ud_verbs_iface_t);
+    uct_ud_verbs_ep_t *ep = ucs_derived_of(ud_ep, uct_ud_verbs_ep_t);
 
-    status = uct_ud_am_common(&iface->super, &ep->super, id, skb_p);
-    if (status != UCS_OK) {
-        return status;
+    if (skb->len >= iface->super.config.max_inline) {
+        uct_ud_verbs_ep_tx_skb(iface, ep, skb);
```
Comment (yosefe) [inline src/uct/ib/ud/verbs/ud_verbs.c]: can unite to one function, and this if would select IBV_SEND_INLINE flag

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/859/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/286/ for details. (Mellanox internal link)

**Comment by yosefe** [comment]
:+1:

**Pull request merged**